src/main/resources/application.conf

```diff
@@ -2,3 +2,14 @@ akka {
   loglevel = INFO
   log-dead-letters-during-shutdown = off
 }
+
+# The server pipeline configuration to capture call metrics.
+squbs.pipeline.server.default {
+  pre-flow = sampleMetricsFlow
```

Comment:
It looks like so.  Not sure how to address it though.  Maybe little bit of explanation in https://github.com/paypal/squbs/blob/master/docs/pipeline.md#pipeline-declaration about post and pre flows, where they get plugged etc..